babel.config.js

```diff
@@ -0,0 +1,10 @@
+// babel.config.js
+module.exports = {
+  presets: [
+    '@babel/env',
+    '@babel/react'
+  ],
+  plugins: [
+    '@babel/plugin-proposal-object-rest-spread',
+  ]
+};
```

Comment:
@ChriLnth wasnt this somewhere else?

Reply:
Just checked, and kept the previous version, I somehow missed it